web-common/src/features/entity-management/resource-invalidations.ts

```diff
@@ -149,16 +149,16 @@ async function invalidateRemovedResource(
   switch (resource.meta.name.kind) {
     case ResourceKind.Source:
     case ResourceKind.Model:
-      queryClient.cancelQueries({
-        predicate: (query) => isProfilingQuery(query, resource.meta.name.name),
-      });
+      // queryClient.cancelQueries({
```

Comment:
We need these since there could be a race condition in request for profile sent and in queue in backend vs the entity being deleted.

Reply:
Same applies to line 158 and line 99 in ModelBody.